.github/ISSUE_TEMPLATE/(1) >2 thema's

```diff
@@ -0,0 +1,14 @@
+---
+name: Candidate Component Issue
+about: Voeg een nieuwe Candidate-component toe aan het NL Design System
+title: "[component] >2 thema's"
```

Comment:
Dit gaat goed, maar de cursor komt hier achter `thema's` terecht bij het aanmaken van het issue. Ik weet niet of we het er gisteren uitgebreid genoeg over gehad hebben maar mijn voorstel was om hier:

">2 thema's {component}" van te maken. Dat is ook in lijn met het voorstel van Jeffrey om `{}` te gebruiken voor variabelen.